tools/rerun/rerun_st.py

```diff
@@ -0,0 +1,146 @@
+RERUN_VERSION = "rerun-cli 0.16.0-alpha.2 [rustc 1.76.0 (07dca489a 2024-02-04), LLVM 17.0.6] x86_64-unknown-linux-gnu main 660463d, built 2024-04-28T12:33:59Z"
+
+
+try:
+    import rerun as rr
+    import rerun.blueprint as rrb
+    # Simple version check
+    result = subprocess.run(["rerun", "--version"], capture_output=True, text=True)
+    if result.returncode == 0:
+        if result.stdout.strip() != RERUN_VERSION.strip():
+          print("YOU NEED TO UNINSTALL RERUN AND THAN START THIS SCRIPT")
+          print("You have "+ result.stdout)
+          print("You need "+ RERUN_VERSION)
+          print("\nHINT: pip uninstall rerun-sdk")
+          exit(0)
+except ImportError:
+    print("Rerun SDK is not installed. Trying to install it...")
+    # subprocess.run([sys.executable, "-m", "pip", "install", "rerun-sdk"])
+    subprocess.run(
+        [sys.executable, "-m", "pip", "install", "--pre", "-f", WHEEL_URL, "--upgrade", "rerun-sdk"],
+        check=True
+    )
+    print("Rerun installed, restarting script")
+    os.execv(sys.executable, [sys.executable] + sys.argv)
+
+
+# Import newest rerun version, replace when rerun version is greater than 0.15.1
+# try:
+#     import rerun as rr
+#     import rerun.blueprint as rrb
+# except ImportError:
+#     print("Rerun SDK is not installed. Trying to install it...")
+#     subprocess.run([sys.executable, "-m", "pip", "install", "rerun-sdk"])
+#     print("Rerun installed, restarting script")
+#     os.execv(sys.executable, [sys.executable] + sys.argv)
+
+
+topics = ['accelerometer', 'androidLog', 'cameraOdometry', 'can', 'carControl', 'carParams', 'carState', 'clocks', 'controlsState', 'deviceState',
+          'driverCameraState', 'driverEncodeIdx', 'driverMonitoringState', 'driverStateV2', 'errorLogMessage', 'gnssMeasurements', 'gpsLocationExternal',
+          'gyroscope', 'initData', 'lateralPlanDEPRECATED', 'lightSensor', 'liveCalibration', 'liveLocationKalman', 'liveParameters', 'liveTorqueParameters',
+          'liveTracks', 'logMessage', 'longitudinalPlan', 'magnetometer', 'managerState', 'mapRenderState', 'microphone', 'modelV2', 'navInstruction',
+          'navModel', 'navThumbnail', 'onroadEvents', 'pandaStates', 'peripheralState', 'procLog', 'qRoadEncodeIdx', 'radarState', 'roadCameraState',
+          'roadEncodeIdx', 'sendcan', 'sentinel', 'temperatureSensor', 'ubloxGnss', 'ubloxRaw', 'uiDebug', 'uiPlan', 'uploaderState',
+          'wideRoadCameraState', 'wideRoadEncodeIdx']
+excluded = ['sentinel']
+is_first_run = True
+
+
+# Log data-(key, value) to rerun
+# Here a property could be excluded from graphing
+def log_data(key, value):
+    rr.log(str(key), rr.Scalar(value))
+
+
+# Log capnp message to rerun
+def log_msg(msg, parent_key=''):
+    stack = [(msg, parent_key)]
+    while stack:
+        current_msg, current_parent_key = stack.pop()
+        if hasattr(current_msg, 'schema') and hasattr(current_msg.schema, 'fields'):
```

Comment:
This doesn't support can, sendcan, liveTracks, logMessage, ubloxRaw, pandaStates, onroadEvents, which I'm sure we want to support at least can, sendcan, liveTracks, pandaStates, and onroadEvents. For the lists, you can just run the `_DynamicListReader` check on the current message